src/components/response/ResponseSwitcher.tsx

```diff
@@ -113,8 +115,10 @@ export function ResponseSwitcher({
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [response.paramCapture, (response as MatrixResponse).questionOptions, (response as SliderResponse).startingValue, response.type, searchParams]);
 
+  const responseDividers = useMemo(() => response.withDivider ?? configInUse?.responseDividers ?? studyConfig.uiConfig.responseDividers, [response, configInUse, studyConfig]);
+
   return (
-    <Box mb={response.withDivider || configInUse.responseDividers ? 'xl' : 'lg'}>
+    <Box mb={response.withDivider || responseDividers ? 'xl' : 'lg'}>
```

Comment:
Seems you already deal with this above on 118.

```suggestion
    <Box mb={responseDividers ? 'xl' : 'lg'}>
```

Reply:
Updated